experiments/findUncoveredAPI.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python3
+
+# Copyright 2020 The GraphicsFuzz Project Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
Oh one more comment: Could you name the file something like `find_uncovered_api.py`? We tend to use this style for Python files.